first-step-spring-autowired/first-step-spring-autowired.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
```

Comment:
No ciężko masz ze mną Rzonsa. Pliki iml to konfiguracja intellij nie wrzucamy tego do repozytorium